fund/qubes-donations-eth-address.txt

```diff
@@ -2,7 +2,7 @@ The Qubes OS Project has one official Ethereum (ETH) donation address:
 
 0xDaa04647e8ecb616801F9bE89712771F6D291a0C
 
-This address is valid on Ethereum network only, it can _not_ receive funds on
+This address is valid only on the Ethereum network. It _cannot_ receive funds on
 other blockchains.
```

Comment:
@marmarek, we still need to clarify two things:

1. Is this address is capable of receiving non-ETH assets (e.g., ERC-20 tokens like USDT and USDC) on Ethereum mainnet?
2. Does "other blockchains" also exclude layer 2 scaling solutions built on top of the Ethereum blockchain, such as Arbitrum?

Reply:
FWIW, the most restrictive version would look something like this:

"WARNING: This address is capable of receiving only native ether (ETH) on
Ethereum Mainnet. Please DO NOT send any other assets or assets on any other
networks to this address, or else your donation may be lost."

Reply:
"Safe supports Ether and all assets that fully comply with the ERC20 standard. This includes assets such as DAI, USDC, UNI, and many others."
Does this answer the question?

Reply:
> "Safe supports Ether and all assets that fully comply with the ERC20 standard. This includes assets such as DAI, USDC, UNI, and many others." - https://help.gnosis-safe.io/en/articles/4970832-supported-asset-types Does this answer the question?

This answers the first question but not the second one.

However, this page appears to answer the second question:

https://help.gnosis-safe.io/en/articles/5267779-i-sent-assets-to-a-safe-address-on-the-wrong-network-any-chance-to-recover

Seems like the answer is: No, we can't accept donations on L2s or sidechains, unless we specifically set up a Gnosis Safe on each of those networks.

We might want to consider doing this for, e.g., Arbitrum at some point, but we can discuss that later.